src/view/films-sort.js

```diff
@@ -0,0 +1,35 @@
+import AbstractView from './abstract';
+import {SortNames} from '../lib/consts';
+
+const createFilmsSortTemplate = () => (
+  `<ul class="sort">
+    <li><a href="#" class="sort__button sort__button--active" data-sort-type="${SortNames.BY_DEFAULT}">Sort by default</a></li>
+    <li><a href="#" class="sort__button" data-sort-type="${SortNames.BY_DATE}">Sort by date</a></li>
+    <li><a href="#" class="sort__button" data-sort-type="${SortNames.BY_RATING}">Sort by rating</a></li>
+  </ul>`
+);
+
+export default class FilmsSort extends AbstractView  {
+  constructor() {
+    super();
+    this._sortChangeHandler = this._sortChangeHandler.bind(this);
+  }
+
+  getTemplate() {
+    return createFilmsSortTemplate();
+  }
+
+  _sortChangeHandler(evt) {
+    if (evt.target.tagName !== 'A') {
+      return;
+    }
+    evt.preventDefault();
+    this._callback.sortTypeChange(evt.target.dataset.sortType);
```

Comment:
во время работы экземпляра класса может оказаться, что в свойстве `sortTypeChange` не указана функция
тогда программа падает с ошибкой.
для придания устойчивости полезно взять на вооружение и использовать такой подход
```suggestion
    const name = EventNames.SORT_TYPE_CHANGE;
    const callbackFunc = this._callback[name];
    if(typeof callbackFunc === 'function'){
      callbackFunc(evt.target.dataset.sortType);
    }
```